### PR TITLE
Release 1.4.1

### PR DESCRIPTION
**Pull request by e5l**
Fixes [KTOR-968](https://youtrack.jetbrains.com/issue/KTOR-968)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -95,7 +128,7 @@
 * Fixed maxFrameSize for chunked frames
 * Fixed URL scheme parser (#1614)
 * Fixed platform detection in `PlatformUtils.IS_NODE` and `PlatformUtils.IS_BROWSER` (#1675)
-* Fixed `Short.highByte` 
+* Fixed `Short.highByte`
```
Comment (Stexxe) [inline CHANGELOG.md:131]: Are changes in this file necessary?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,36 @@
+# 1.4.1
+> Published 23 Sep 2020
+
+* OkHttp: Can't reuse same HttpRequestBuilder for different network clients (KTOR-949)
```
Comment (Stexxe) [inline CHANGELOG.md:4]: Will issues' ID be converted to Youtrack links?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,36 @@
...
+* Empty body in response using macosx64 target (KTOR-479)
```
Comment (Stexxe) [inline CHANGELOG.md:5]: `x` should be capitalized

**Review by Stexxe — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (e5l) [reply CHANGELOG.md:131]: thanks, will fix

File: CHANGELOG.md (hunk 2 above)
Comment (e5l) [reply CHANGELOG.md:4]: I will try to make this

**Review by rsinukov — APPROVED** [review]

**Review by Stexxe — APPROVED** [review]

**Review by Ololoshechkin — APPROVED** [review]

**Review by cy6erGn0m — APPROVED** [review]

**Review by hhariri — APPROVED** [review]

**Pull request merged**